dynawo/sources/ModelicaCompiler/Scripts_OMC_1_13_2/factory.py

```diff
@@ -1418,6 +1431,67 @@ def prepare_for_sety0(self):
 
         # convert native boolean variables
         convert_booleans_body ([item.get_name() for item in self.list_all_bool_items], self.list_for_sety0)
+
+    ##
+    # prepare the variables for setY0External methods
+    # @param self : object pointer
+    # @return
+    def prepare_for_sety0external(self):
+        # We concatenate system vars and discrete vars
+        list_vars = itertools.chain(self.list_vars_syst, self.list_all_vars_discr, self.list_vars_int, self.reader.list_complex_const_vars)
```

Comment:
Please remove the discrete from this list and adapt the comment

Reply:
OK

Reply:
self.list_vars_syst is enough

Reply:
OK